api/src/main/java/org/eclipse/microprofile/metrics/Metadata.java

```diff
@@ -92,10 +92,8 @@
 
     Optional<String> getUnit();
 
-    boolean isReusable();
-
     /**
-     * Returns a new builder
+     * Re   turns a new builder
```

Comment:
ok, is this one a trick to see if reviewers are paying attention? ;-)

Reply:
Exactly!